src/main/java/com/microsoft/graph/httpcore/middlewareoption/TelemetryOptions.java

```diff
@@ -0,0 +1,38 @@
+package com.microsoft.graph.httpcore.middlewareoption;
+
+import java.math.BigInteger;
+import java.util.UUID;
+
+public class TelemetryOptions {
+
+	public static final String NONE_FLAG = "0x00000000";
+	public static final String REDIRECT_HANDLER_ENABLED_FLAG =  "0x00000001";
+	public static final String RETRY_HANDLER_ENABLED_FLAG = "0x00000002";
+	public static final String AUTH_HANDLER_ENABLED_FLAG = "0x00000004";
+	public static final String DEFAULT_HTTPROVIDER_ENABLED_FLAG = "0x00000008";
+	public static final String LOGGING_HANDLER_ENABLED_FLAG = "0x00000010";
```

Comment:
how will you distinguish between
DEFAULT_HTTPROVIDER_ENABLED_FLAG  && RETRY_HANDLER_ENABLED_FLAG 
vs LOGGING_HANDLER_ENABLED_FLAG. Lets discuss these scenarios.

Reply:
DEFAULT_HTTPROVIDER_ENABLED_FLAG is 1000
RETRY_HANDLER_ENABLED_FLAG is 10
1000 | 10 = 'a'(hexa) and 1010(binary)
LOGGING_HANDLER_ENABLED_FLAG is 10(hexa) and 10000(binary)